packages/react/src/__tests__/createReactClassIntegration-test.js

```diff
@@ -365,7 +365,7 @@ describe('create-react-class-integration', () => {
       expect(() => Component()).toThrow(),
     ).toErrorDev(
       'Warning: Something is calling a React component directly. Use a ' +
-        'factory or JSX instead. See: https://fb.me/react-legacyfactory',
+        'factory or JSX instead. See: https://reactjs.org/warnings/legacy-factories.html',
```

Comment:
This change causes the test to fail, because the `create-react-class` package has not been changed. (Indeed that class has not been updated in 3 years.)

I think it's fine to leave the fb.me reference in this integration test. Let's revert the change to this file?

Reply:
@bvaughn I'm reverting the change but I don't understand how is an error string causing the test to fail.
How can we approach this ?